src/test/java/com/karankumar/booksapi/model/BookTest.java

```diff
@@ -30,13 +30,26 @@ void notAcceptNullTitle() {
 
         // when and then
         assertThatExceptionOfType(NullPointerException.class)
-                .isThrownBy(() -> new Book(null, new Author[] { author} ));
+                .isThrownBy(() -> new Book(null, new Author[] { author}, Language.ENGLISH));
     }
 
     @Test
     @DisplayName("throw a Null Pointer Exception on an attempt to create with a null Author")
     void notAcceptNullAuthor() {
         assertThatExceptionOfType(NullPointerException.class)
-                .isThrownBy(() -> new Book("The Hobbit", null));
+                .isThrownBy(() -> new Book("The Hobbit", null, Language.ARABIC));
     }
+    
+    
+    @Test
+    @DisplayName("throw a Null Pointer Exception on an attempt to create with a null language")
+    void notAcceptNullLanguage() {
+        // given
+        Author author = new Author("J.K.", "Rowling");
+
+        // when and then
+        assertThatExceptionOfType(NullPointerException.class)
+            .isThrownBy(() -> new Book(null, new Author[] { author}, null));
```

Comment:
Could you please add a title here rather than using `null`? Otherwise, there are two reasons why this statement could throw a null pointer exception